e2e_test/batch/basic/temp.slt.part

```diff
@@ -0,0 +1,22 @@
+statement ok
```

Comment:
What's this file for?

Reply:
Slipped in there by accident. Removed it 👍